utilities/check_mrTrix_Version.m

```diff
@@ -0,0 +1,19 @@
+function mrTrixVersion = check_mrTrix_Version
```

Comment:
Furthermore, it doesn't make sense, because your previous function
check_mrtrix was doing the same but for many other functions. I am just
checking mrconvert. If it is not properly installed, it will fail, but it
will be the same for all the functions.

Check that you don't have a different problem, did you change your machine
recently?
I had a problem in linux with this as well, the call from the command line
would work but the call from matlab not, because matlab is using its own
libraries.
Can you send me the output of this please?
system('mrconvert -version')
system('gcc -v')

thanks!

On Tue, Jul 12, 2016 at 1:56 AM Garikoitz Lerma-Usabiaga <
garikoitz@gmail.com> wrote:

> Yes, I wrote a note. I don't have a problem with it in OSX, and in two
> different linux boxes (both Red Hat), one normal linux box and the other
> one part of a cluster. Are you using ubuntu?
>
> This is the note:
>
> % Jason, I don't have the AFQ_mrtrix_set_ld_path problem (in linux/OSX), I
> get rid of it with
>
> % the matlab call itself. Furthermore, it would be circular, since that
> function
>
> % is using the version information as well. We could us a function call
> used in
>
> % both versions. Please edit this as it is working for you and I
> % will override it in my version if necessary.
>
> I don't understand what your problem is, can you send me the error please?
>
> On Tue, Jul 12, 2016 at 12:56 AM dstrodtman <notifications@github.com>
> wrote:
>
>> In utilities/check_mrTrix_Version.m
>> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r70352285>:
>>
>> > @@ -0,0 +1,19 @@
>> > +function mrTrixVersion = check_mrTrix_Version
>>
>> This doesn't look like it has been fixed. For this function to run
>> properly we need to first call AFQ_mrtrix_set_ld_path. However
>> AFQ_mrtrix_set_ld_path takes as an input, the output of the present
>> function. Gari, did this work for you on linux?
>>
>> —
>> You are receiving this because you modified the open/close state.
>>
>>
>> Reply to this email directly, view it on GitHub
>> <https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r70352285>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AEWQV5c4XAECeJhde8WXaJsfh1eYoxrYks5qUsoggaJpZM4JGbEU>
>> .
>>
>